exercises/concept/cars-assemble/.meta/design.md

```diff
@@ -25,3 +25,17 @@ This exercise's prerequisites Concepts are:
 
 - `basics`: know how to define methods.
 - `booleans`: know how to use boolean operators.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `actionable`: If the student did not reuse the implementation of the `productionRatePerHour` method in the `workingItemsPerMinute` method, instruct them to do so.
+- `informative`: If the solution is hardcoding the value `221`, inform the student that it could be great to extrapolate this value to a `private final` field to make a more readable code.
```

Comment:
```suggestion
- `informative`: If the solution is repeatedly hard-coding the value `221`, inform the student that they could store this value in a field to make the code easier to maintain.
```

Two things:
- The Java track doesn't really teach the concept of a `final` field or variable at this time, so I'm not sure the analyzer should point students towards it explicitly.
- The comment _could_ link to an explanation of the DRY principle if you think that's a good idea (I'm not 100% sure myself so I leave it up to you).

Reply:
Hmm I don't know if it would be useful to link it in the design.md file maybe in the comment itself when we build the analyzer, for me how you rephrase it seems ok!

Reply:
Yeah I meant for the comment copy to link to it.